x-pack/plugin/esql/qa/testFixtures/src/main/resources/floats.csv-spec

```diff
@@ -592,3 +592,10 @@ s:double | emp_no:integer | salary:integer  | salary_change:double
 -1.0     | 10004          | 36174           | [-0.35, 1.13, 3.65, 13.48]
 -1.0     | 10005          | 63528           | [-2.14, 13.07]
 ;
+
+scalb
```

Comment:
Please add a few more tests here, eg. with:
- an index as the source of the query (multiple records in the result)
- both foldable and non-foldable parameters
- usage of the function in a WHERE condition